README.md

```diff
@@ -1 +1,63 @@
-# Boltdir
+# Skeleton Bolt project directory
+
+Bolt is a Ruby command-line tool for executing commands, scripts, and tasks on remote systems using SSH and WinRM.
+
+* Executes commands on remote *nix and Windows systems*.
+* Distributes and execute scripts, such as Bash, PowerShell, Python.
+* Scales to more than 1000 concurrent connections.
+* Supports industry standard protocols (SSH/SCP, WinRM/PSRP) and authentication methods (password, publickey).
+
+> Installing bolt from a gem is not recommended since core modules will not be available. Please [install bolt](https://puppet.com/docs/bolt/latest/bolt_installing.md) as a package
+
+This repository is an example skeleton Bolt Project Directory or `Boltdir`. A `Boltdir` contains all of the configuration, code, and data loaded by Bolt. Read more about the project directory structure at the following [link.](https://puppet.com/docs/bolt/latest/bolt_project_directories.html#project-directories)
+
+## Directory structure
```

Comment:
I think this should just be a link to the docs page, that way there's only 1 source of truth.

Reply:
I think it is important to leave this as is. I link to the docs about choosing an appropriate directory structure above. In terms of a "source of truth" this shows the directory structure of this repository. So the only change would be if we added or removed a file from this project.